query-node/mappings/src/content/nft.ts

```diff
@@ -285,6 +285,8 @@ async function finishAuction(
 
   // load winner member
   const winnerMemberId = winningBid.bidder.id
+  // load bid amount for which NFT is bought
+  const boughtPrice = (auction.topBid as Bid).amount
```

Comment:
This needs to use `winningBid` as well, in open auction the lowest bid can be chosen as the winner as well